network/stations.csv

```diff
@@ -1631,6 +1631,7 @@ PKGZ,EC,Pakihiroa,-37.8664,178.0805,405,,WGS84,2010-06-22T00:00:00Z,9999-01-01T0
 PKHE,XX,Pakihiroa,-37.859023554,178.093517735,200,,WGS84,1994-08-01T05:33:00Z,1994-12-14T03:38:00Z
 PKIS,SX,Pukaki Dam,-44.190967535,170.146940461,494,,WGS84,2001-06-14T00:00:00Z,2015-06-30T23:59:59Z
 PKRC,XX,Pukeroa,-38.480729926,175.540484557,600,,WGS84,2001-01-18T13:22:00Z,2001-04-13T15:35:00Z
+PKTZ,TP,Puketapu Road,-38.672315,175.804593,514.959,,WGS84,2024-07-16T22:06:00Z,9999-01-01T00:00:00Z
```

Comment:
coordinates and elevation should be the same with the entry in `sites.csv`?

Reply:
Forgot to add new coordinates to this file 🤦 updated now